skyportal/handlers/api/internal/plot.py

```diff
@@ -7,22 +7,36 @@
 class PlotPhotometryHandler(BaseHandler):
     @auth_or_token
     def get(self, source_id):
```

Comment:
Do we need an OpenAPI spec for an internal API?

Reply:
We typically don't have them, no